crates/kornia-3d/src/pointcloud.rs

```diff
@@ -49,6 +51,38 @@ impl PointCloud {
     pub fn normals(&self) -> Option<&Vec<[f64; 3]>> {
         self.normals.as_ref()
     }
+
+    /// Convert a point from [f64; 3] to Vec3.
+    fn point_to_vec3(point: &[f64; 3]) -> Vec3 {
```

Comment:
this doesn't have to belong to PointCloud and can be implemented via the the Into/from trait